tools/build/docker/s6/s6-rc.d/lanraragi/run

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+cd /home/koyomi/lanraragi/
+export HOME=/home/koyomi
+exec s6-setuidgid koyomi perl /home/koyomi/lanraragi/script/launcher.pl -f /home/koyomi/lanraragi/script/lanraragi
```

Comment:
Does this mean the older `services.d/lanraragi` and `services.d/redis` folders are not to be used anymore? I'm guessing the s6 structure changed between versions. 

I was going to say we could remove those in that case, but since I have to maintain a legacy alpine 3.12 build for the time being I can't lol 🫠  

Comments to explain that difference would be much appreciated though!